src/components/structures/SpaceRoomView.tsx

```diff
@@ -614,7 +614,20 @@ const SpaceAddExistingRooms = ({ space, onFinished }) => {
             filterPlaceholder={_t("Search for rooms or spaces")}
             onFinished={onFinished}
             roomsRenderer={defaultRoomsRenderer}
-            spacesRenderer={defaultSpacesRenderer}
+            spacesRenderer={() => (
```

Comment:
sorry I didn't realise which call-site this was, I think there should be no spacesRenderer here and it should just show rooms to add instead

Reply:
I think so... done